packages/machine-extractor/src/__tests__/options.test.ts

```diff
@@ -2,24 +2,34 @@ import * as t from '@babel/types';
 import { extractMachinesFromFile } from '..';
 
 describe('Options', () => {
-  it('Should handle functions declared as ObjectMethod', () => {
+  it('Should extract object methods and functions declared as object methods', () => {
```

Comment:
this test title is confusing to me now because it suggests that all functions in it should be declared as object methods but only `setContext` is a method